napari/layers/labels/_labels_key_bindings.py

```diff
@@ -1,3 +1,5 @@
+import typing
```

Comment:
Maybe 
```suggestion
from typing import cast
```

If this is the only place where it is used, perhaps just import the function